ninja/write_ninja.py

```diff
@@ -0,0 +1,65 @@
+# Experimental ninja
+
+from ninja import ninja_syntax
```

Comment:
I presume you forgot to check in the repo the ninja_syntax.py module from https://github.com/ninja-build/ninja/blob/master/misc/ninja_syntax.py

There's also a `ninja_syntax` module that one can pip install from PyPI but I don't know if it's in sync with upstream or maintained (last updated in 2017).

Reply:
ninja_syntax comes along with ninja.

Reply:
It comes with ninja